doc/pprof.md

```diff
@@ -105,10 +105,50 @@ number of values - 1) or the name of the sample value.
 
 Sample values are numeric values associated to a unit. If pprof can recognize
 these units, it will attempt to scale the values to a suitable unit for
-visualization. The `unite=` option will force the use of a specific unit. For
+visualization. The `unit=` option will force the use of a specific unit. For
 example, `sample_index=sec` will force any time values to be reported in
 seconds. pprof recognizes most common time and memory size units.
 
+### Tag Filtering
```

Comment:
I'd keep it second level header, just to avoid too much nesting. Also put an empty line after it, as a nit.